src/record.ts

```diff
@@ -16,10 +16,10 @@
 
 'use strict';
 
-var arrify = require('arrify');
-var common = require('@google-cloud/common');
-var extend = require('extend');
-var format = require('string-format-obj');
+import {util} from '@google-cloud/common';
+import extend from 'extend';
+import arrify from 'arrify';
```

Comment:
Half-hearted. I would have expected that you can drop the dependency on arrify now.